controller/page.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+page('/', showHome());
+page('/about', showAbout());
```

Comment:
Two things - you don't invoke functions when you pass them to page. Also, showHome and showAbout aren't globally available. You need to attach them to something being exported from an IIFE (via the window) and then reference them as methods on that object. Check out the way you did this with your Project global object.